articles/tech-resources/sparkpost-event-metrics-definitions.md

```diff
@@ -32,56 +32,57 @@ Individual events are provided via the webhook data stream and can be retrieved
+| Rendered/Opens                   | Total renderings of a message                                                                                                               | Sum (messages opened)                                              | <ul><li>The total number of times a **message** is displayed (whether fully opened or within the preview pane) and recorded using a tracking pixel for a unique subscriber address. If the Recipient opens the email multiple times, one email render is counted for each occurrence. Non-distinct.</li><li>This metric applies to HTML formatted emails only.</li></ul>                                                                                |
+| Unique Confirmed Opens (approx.) | The estimated number of messages, within a 5% variance, that were either rendered or had at least one link selected.                        |                                                                    |                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| Unique Confirmed Opens           | Total number of messages that were either rendered or had at least one link selected. (This metric may take longer to display.)             | Sum (unique messages opened or clicked)                            | The unique number of **unique messages** opened (whether fully opened or within the preview pane) and recorded using a tracking pixel OR if images are blocked and the user clicks any link including the unsubscribe link. Includes Opened OR Clicked unique messages.                                                                                                                                                                                 |
+| Open Rate (approx.)              | Open Rate using the Unique Confirmed Opens (approx.) metric                                                                                 | Percentage (Unique Confirmed Opens (approx.) / Accepted)           |                                                                
```

Comment:
This and the following are the only new lines, others were table format fixes. (Prettier handled that for me)